project/plugins.sbt

```diff
@@ -1,11 +1,11 @@
 logLevel := Level.Warn
 
-resolvers += "bintray-spark-packages" at "https://dl.bintray.com/spark-packages/maven/"
+resolvers += Resolver.bintrayIvyRepo("s22s", "sbt-plugins")
 
-addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.6")
+addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.7-astraea.1")
 
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.2")
```

Comment:
This change was necessary because, it seems there is no 1.6.0 version for scala 2.12

I don't think this change is problematic at all. Should be OK for both Scala versions.

Reply:
I think the work from scalafmt could be a good replacement. 
https://scalameta.org/scalafmt/docs/installation.html#sbt

```
// Note, does not support sbt 0.13, only sbt 1.0.
addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
```

The only risk I see is that it has to be used with sbt 1.X.